pkgs/development/python-modules/mitmproxy/default.nix

```diff
@@ -96,21 +87,39 @@ buildPythonPackage rec {
   ];
 
   checkInputs = [
-    asynctest
     beautifulsoup4
-    flask
     glibcLocales
     hypothesis
     parver
-    pytest
     pytest-asyncio
+    pytest-timeout
+    pytest-xdist
+    pytestCheckHook
     requests
   ];
 
+  doCheck = !stdenv.isDarwin;
+
+  postPatch = ''
```

Comment:
Please don't move pieces of code around when there is no need to.

Reply:
`nixpkgs-hammer` has complained about the ordering.

Reply:
Fine for me when we change bigger parts anyway.